csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -12,14 +13,26 @@
   #include <hipcub/hipcub.hpp>
 #endif
 
+#define USE_ROCM
+
+// Explicitly set the rounding mode to nearest
+template <typename T>
+auto __device__ to_nearest(T x) {
+  auto const mode = std::fegetround();
+  std::fesetround(FE_TONEAREST);
+  auto const result = std::nearbyint(x);
+  std::fesetround(mode);
+  return result;
+}
```

Comment:
Is this expensive to do at the granularity of a single instruction? Probably best to lift it out of the inner loop

Reply:
Yeah that's a good call

Reply:
Removed, HIP only supports `tonearest`. Added a comment